src/DssSpell.sol

```diff
@@ -45,16 +52,164 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_1000000003022265980097387650RATE = ;
+    uint256 internal constant EIGHT_PCT_RATE               = 1000000002440418608258400030;
+    uint256 internal constant EIGHT_PT_TWO_FIVE_PCT_RATE   = 1000000002513736079215619839;
+    uint256 internal constant EIGHT_PT_SEVEN_FIVE_PCT_RATE = 1000000002659864411854984565;
+    uint256 internal constant NINE_PCT_RATE                = 1000000002732676825177582095;
+    uint256 internal constant NINE_PT_TWO_FIVE_PCT_RATE    = 1000000002805322428706865331;
+    uint256 internal constant NINE_PT_FIVE_PCT_RATE        = 1000000002877801985002875644;
+    uint256 internal constant NINE_PT_SEVEN_FIVE_PCT_RATE  = 1000000002950116251408586949;
+    uint256 internal constant TEN_PT_TWO_FIVE_PCT_RATE     = 1000000003094251918120023627;
 
     // ---------- Contract addresses ----------
+    GemAbstract internal immutable MKR = GemAbstract(DssExecLib.mkr());
+
+    // ---------- Dss-Cron Update ----------
+    address internal immutable CRON_SEQUENCER  = DssExecLib.getChangelogAddress("CRON_SEQUENCER");
+    address internal immutable CRON_D3M_JOB    = DssExecLib.getChangelogAddress("CRON_D3M_JOB");
```

Comment:
This is not a problem in my opinion, but just want to flag this for visibility. Fetching `CRON_SEQUENCER` and current `CRON_D3M_JOB` addresses from the chainlog differs from the instructions that we're given with hardcoded addresses (e.g. `sequencer.removeJob(0x1Bb799509b0B039345f910dfFb71eEfAc7022323)`). Although the addresses and ultimately the results of those operations are the same at the moment, they might change in case another spell modifies them in-between. As it is highly unlikely, I would keep it as is.

Reply:
I believe such instructions are outdated, because all `CRON_*` addresses used to **NOT** be in the chainlog.

In reality, this would be a problem for every single spell that fetches addresses from the chainlog, as they would get the value at deployment time instead. However, I'd say chances are pretty minimal, because if there are any outstanding spells, the crafter would know &ndash; since the process mandates it &ndash; can could take the appropriate measures (i.e.: hardcoding the address being modified in the chainlog).

Reply:
I agree that the instruction could've been better in this case, e.g. `Update CRON_D3M_JOB in CRON_SEQUENCER to the new address 0x2Ea4aDE144485895B923466B4521F5ebC03a0AeF`, but I don't feel strong about requesting a change – it's clear what needs to be done and chainlog update is always a technical nuance that doesn't need an explicit instruction